packages/@aws-cdk/core/lib/arn.ts

```diff
@@ -135,39 +135,18 @@ export class Arn {
    *      components of the ARN.
    */
   public static parse(arn: string, sepIfToken: string = '/', hasName: boolean = true): ArnComponents {
```

Comment:
This applies to line 121 (which GH doesn't let me add a comment at)

Update the statement here recommending the use of `extractResourceName()`